frontend/js/restqueries.js

```diff
@@ -0,0 +1,82 @@
+try {
+	var staticURL = "http://ec2-34-208-19-115.us-west-2.compute.amazonaws.com:18080/character/";
+	var playerURL = "OtherTestChar";
+	var DnDObj = getUser();
+	//getJSONField("player");
+}
+catch (err) {
+	console.log(err);
+}
+//getJSONField("name");
+populateDNDFields();
+
+function getUserText() {
+	var xhttp = new XMLHttpRequest();
+	xhttp.open("GET", staticURL + playerURL, false); //find a way to set async to true and have it return when it has something
+	xhttp.setRequestHeader("Content-type", "text/plain");
+	xhttp.send();
+	var response = JSON.stringify(xhttp.responseText);
+	console.log(response + " GetUserText() Response");
+	return response;
+}
+
+function getUser() {
+	var xhttp = new XMLHttpRequest();
+	xhttp.open("GET", staticURL + playerURL, false); //find a way to set async to true and have it return when it has something
+	xhttp.setRequestHeader("Content-type", "text/plain");
+	xhttp.send();
+	var response = JSON.parse(xhttp.responseText);
+	console.log(response + " GetUser() Response");
+	return response;
+}
+//Gets JSON field and puts the value into a field of THE SAME NAME (could be made better such that the json field and the html field would be better differentiated)
+function getJSONField(JSONFieldName) {
+	var fieldName = DnDObj[JSONFieldName];
```

Comment:
Not sure, but I think this might crash if DnDObj doesn't contain the key specified. You might have to check if the dict contains the key first before calling it.